script/electron-builder-linux.yml

```
@@ -1,36 +1,15 @@
-productName: 'GitHubDesktop'
-artifactName: '${productName}-${os}-${arch}-${version}.${ext}'
+artifactName: 'GitHubDesktop-${os}-${arch}-${version}.${ext}'
 linux:
   category: 'GNOME;GTK;Development'
   packageCategory: 'GNOME;GTK;Development'
-  icon: 'app/static/logos'
+  icon: 'app/static/linux/logos'
+  mimeTypes:
+    - x-scheme-handler/x-github-client
+    - x-scheme-handler/x-github-desktop-auth
+    # workaround for handling OAuth flow until we figure out what we're doing
+    # With the development OAuth details
+    # see https://github.com/shiftkey/desktop/issues/72 for more details
+    - x-scheme-handler/x-github-desktop-dev-auth
   target:
-    - deb
-    - rpm
     - AppImage
   maintainer: 'GitHub, Inc <opensource+desktop@github.com>'
```

This is probably something I should update:

```suggestion
  maintainer: 'Brendan Forster<github@brendanforster.com>'
```